ast/ddl.go

```diff
@@ -2037,12 +2038,16 @@ const (
 	SequenceCache
 	SequenceNoCycle
 	SequenceCycle
+	// SequenceRestart is only used in alter sequence statement.
+	SequenceRestart
 )
 
 // SequenceOption is used for parsing sequence option from SQL.
 type SequenceOption struct {
 	Tp       SequenceOptionType
 	IntValue int64
+	// SequenceRestart is only used for restart option in alter sequence statement.
+	NoValue bool
```

Comment:
the field name and comment does not match.

(maybe make `SequenceRestart` and `SequenceRestartWith` two different `SequenceOptionType` to avoid this extra field 🤔)

Reply:
> maybe make `SequenceRestart` and `SequenceRestartWith` two different SequenceOptionType to avoid this extra field

Agreed.

Reply:
good